src/hmem.c

```diff
@@ -114,6 +114,13 @@ static struct ofi_hmem_ops hmem_ops[] = {
 	},
 };
 
+bool ofi_hmem_is_interface_initialized(enum fi_hmem_iface iface)
+{
+	assert(iface >=0 && iface < FI_HMEM_MAX);
+
+	return hmem_ops[iface].initialized;
```

Comment:
It takes more effort to call this function than it does to just access the array inline from the 1 location that calls it.  Please just use the field directly.

Reply:
Fixed in new revision. Note I kept the name `hmem_ops` because it is a smaller change. Let me know if you prefer to add an `ofi_` prefix for consistency.